### ISSUE TITLE
Respect editorconfig instead needing to use `--use-tabs`

### ISSUE THREAD

**Issue by allejo** [cameronhunter/prettier-package-json#31]
Instead of relying on the user to manually use `--use-tabs` or `useTabs`, can this tool respect an existing `.editorconfig`'s configuration as prettier itself, does?

**Comment by cameronhunter** [comment]
Good idea. Is there a package that will parse prettier and editorconfig files that can be used?

**Comment by allejo** [comment]
> Prettier uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) for configuration file support.
>
> Source: https://prettier.io/docs/en/configuration.html

There doesn't seem to be a "one package fits all" for this scenario. Prettier uses a combination of cosmiconfig for its own configuration file and [editorconfig](https://www.npmjs.com/package/editorconfig) for [its handling of `.editorconfig` files](https://github.com/prettier/prettier/blob/24d39a906834cf449304dc684b280a5ca9a0a6d7/src/config/resolve-config-editorconfig.js#L5).

**Comment by cameronhunter** [comment]
I recently added `cosmiconfig`, I can look into `editorconfig`. Of course, contributions are welcome too!
